tests/test_state.py

```diff
@@ -415,7 +415,7 @@ def test_statistics(usdc, weth_usdc, aave_usdc, start_ts):
     trader.buy(weth_usdc, Decimal(0.1), 1700)
     trader.buy(aave_usdc, Decimal(0.5), 200)
 
-    update_statistics(datetime.datetime.utcnow(), state.stats, portfolio)
+    update_statistics(datetime.datetime.utcnow(), state.stats, portfolio, ExecutionMode.backtesting.value)
```

Comment:
Because it is enum, you can pass it just as `ExecutionMode.backtesting`, no `.value` needed